examples/next/tutorial/tailwind.config.mjs

```diff
@@ -0,0 +1,17 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: [
+    "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
```

Comment:
Is this required or can we simplify?

Reply:
We need the `pages` one, but I'll remove the other two to clean things up a bit.